### DISCUSSION TITLE
How can I connect the create3 to a wifi network that uses WPA2-Enterprise, meaning user and password to connect, not just the password

### DISCUSSION THREAD

**Discussion by tonypg39** [iRobotEducation/create3_docs#542] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

G.4.1

### Which RMW is your robot running?

CycloneDDS

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

I'm using a Turtlebot4, and I managed to connect the RPi 4 to the network by configuring the network interface, I don't know in what way I can do it fo the Create3 tho, any advice appreciated!
Thanks

**Comment by shamlian** [comment]
Unfortunately, the Create 3 robot's Wi-Fi connection flow does not support WPA2-Enterprise. Some customers have gotten around this by using MAC address whitelisting, and others have bridged the network connection through another single board computer. I believe this latter approach can be done on the Turtlebot 4.
